docs/source/guides/network_node/staking_guide.rst

```diff
@@ -198,19 +207,19 @@ Next, initialize Clef with your chosen password to encrypt the master seed:
     Password:
 
 
-Running Clef for Goerli
-***********************
+Running Clef
```

Comment:
Continuing the above, this doc includes both clef and local signing approaches, which makes it .... not really a guide... rather... half-guide-half-reference.  IMO this doc will be better suited broken up into at least  two separate docs.  This may be beyond the scope of this PR, but it's clear as a reader that this doc is a bit to dense for a smooth follow-along experience.

Reply:
✔️  - Filed #1952